README.md

```diff
@@ -1,46 +1,11 @@
 # ansible-compat
```

Comment:
I think I'd want the readme to be partially included into the docs front page to avoid duplication as much as possible.

Reply:
We cannot really include it because for few reasons:
* it would include phrase that does not make much sense, pointing to the location of official documentation (would be kind recursive to display it on the official docs page)
* not sure if displaying badges is desirable in published doc but if you think so, ok
* header indentation/level would mismatch and we would get an error while building

I make it easier to progress on that just push extra fixes to that branch. I will avoid forced pushes. I have no strong feeling around how we organize it.

Reply:
That's why I said *partially*. There's start-at, start-after and so on. You should only reuse parts that makes sense to reuse.

Reply:
There is nothing reusable inside root `README.md` file.